Gemfile.lock

```diff
@@ -547,18 +561,19 @@ GEM
     sitemap_generator (6.0.2)
       builder (~> 3.0)
     smart_properties (1.17.0)
-    solargraph (0.48.0)
+    solargraph (0.49.0)
```

Comment:
Is there a reason why solargraph is a gem installed in this project. AFAIK this is a language server and should be installed globally on a developers system

Reply:
Yes. Reason:
>  If your project uses Bundler, the most comprehensive way to use your bundled gems is to bundle Solargraph.
https://marketplace.visualstudio.com/items?itemName=castwide.solargraph